tests/cypress/latest/e2e/capd_rke2_fleet_clusterclass.spec.ts

```diff
@@ -0,0 +1,117 @@
+/*
+Copyright © 2022 - 2023 SUSE LLC
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import '~/support/commands';
+import * as cypressLib from '@rancher-ecp-qa/cypress-library';
+import { skipClusterDeletion } from '~/support/utils';
+
+Cypress.config();
+describe('Import CAPD RKE2 Class-Cluster', { tags: '@short' }, () => {
+  let clusterName: string
+  const timeout = 600000
+  const classNamePrefix = 'docker-rke2'
+  const repoUrl = 'https://github.com/rancher/rancher-turtles-e2e.git'
+  const path = '/tests/assets/rancher-turtles-fleet-example/capd/rke2/class-clusters'
+  const branch = 'capd-fleet'
```

Comment:
Reminder to change this to main.
```suggestion
  const branch = 'main'
```